data/core/units/dwarves/Berserker.cfg

```diff
@@ -9,10 +9,10 @@
     movement_type=dwarvishfoot
     movement=5
     experience=100
+    {AMLA_DEFAULT}
     level=2
     alignment=neutral
-    advances_to=null
-    {AMLA_DEFAULT}
+    advances_to=Dwarvish Bloodaxe
```

Comment:
Hey @Dalas121, the Dwarf Bloodaxe is inside the THoT campaign, but this is modifying the core Dwarven Berserker unit. There might be unit type not found error, if used anywhere outside THoT

Reply:
Oops, fixed, thanks!